tests/integration/echo/echo_test.go

```diff
@@ -36,126 +37,130 @@ var (
 
 // TODO(sven): Add additional testing of the echo component, this is just the basics.
 func TestEcho(t *testing.T) {
-	framework.Run(t, func(ctx framework.TestContext) {
-		g := galley.NewOrFail(t, ctx, galley.Config{})
-		p := pilot.NewOrFail(t, ctx, pilot.Config{
-			Galley: g,
-		})
+	framework.
+		NewTest(t).
+		// TODO(https://github.com/istio/istio/issues/13810)
+		Label(label.Flaky).
```

Comment:
Only the sidecar test is flaking ... and only on Kube. Can you reduce the scope of this so that as much will run on presubmit as possible?

Reply:
Updated. PTAL. There is no way to discern between Kube & Native, in terms of flakes (or rather labels) at the moment.